addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -158,6 +193,58 @@ weSnippetEditor.SnippetsMenu.include({
         this._super(...arguments);
         this.$('.o_we_customize_theme_btn').toggleClass('active', tab === this.tabs.THEME);
     },
+    /**
+     * Returns the animated text element wrapping the selection if it exists.
+     * 
+     * @private
+     * @param {Node} editable
+     */
+    _getAnimatedTextElement(editable) {
+        const animatedTextNode = getTraversedNodes(editable).find(n => $(n.parentElement).closest(".o_animated_text").length);
+        return $(animatedTextNode).closest('.o_animated_text');
```

Comment:
Should really not use jquery anymore  for this kind of stuff which makes the code less readable. There is a closest method in javascript.

Reply:
`animatedTextNode.closest('.o_animated_text');`
closest() doesnt works because `animatedTextNode` is a text node, so I'm not sure if it's worth changing this code.

Reply:
In the first line, you use `n.parentElement`, in the second you do not, why is that? :thinking:

Reply:
Actually I dont need .parentElement here with jquery, it's useless ..